Makefile

```diff
@@ -130,7 +130,7 @@ endif
 
 .PHONY: test-cli
 test-cli: container
-	IMAGE=$(REGISTRY)/metrics-server-$(ARCH):$(CHECKSUM) EXPECTED_VERSION=$(GIT_TAG) ./test/test-cli.sh
+	IMAGE=$(REGISTRY)/metrics-server-$(ARCH):$(CHECKSUM) EXPECTED_ARCH=$(ARCH) EXPECTED_VERSION=$(GIT_TAG) ./test/test-cli.sh
```

Comment:
Can you rename script to ./test/test-image.sh ?

Reply:
This code will be run once and test only default arch. Maybe we could add a target test-cli-all that builds all architecures and tests them.

Reply:
What would you say of something like this?

```Makefile
.PHONY: test-image-all
test-image-all:
	@for arch in $(ALL_ARCHITECTURES); do ARCH=$${arch} $(MAKE) test-image; done
```

Reply:
lg